bcgov/mds
# Assign Project Lead and Update Project Description Status

# Main
- Mixed two tickets together as they had several dependencies MDS-4050 and MDS-4088
- Ability to assign a project lead to the project description
- Ability to update the status of a project description. Added new status, and inactivated a few. Added an alias column as the status description is displayed differently in MineSpace.
- Updated the MineSpace side to only update the status on submission and draft, otherwise persist the current status


# Other

- N/A

# How to test
Assigning a project lead
Create a project description in MineSpace -> submit -> Navigate to the Core side and click 'open' on the table -> Scroll down to the 'Project Contacts' section, and click 'edit' next to the Project Lead field. 
Ensure that you have project leads in your db -> select a project lead from the dropdown, and click 'update'. The status should also change to be "Assigned"

Update the status or project stage
Create a project description in MineSpace -> submit -> Navigate to the Core side and click 'open' on the table
Ensure that the 'First submitted date" is populated  on the table -> the first field is the "Project stage" -> click "edit" and select the status you'd like to update the project description to.


# Notes
![Screen Shot 2022-02-08 at 1 08 55 PM](https://user-images.githubusercontent.com/25125247/153075959-34c2553a-cb3b-423f-bb18-c45cac74b34f.png)

![Screen Shot 2022-02-08 at 1 09 47 PM](https://user-images.githubusercontent.com/25125247/153076076-778e9ff4-c6a2-4af5-b078-0bfe1b04e932.png)

if "Under Review - with proponent" or "Under Review - with reviewer" are selected, the mineSpace side displays only "Under Review"
MineSpace:
![Screen Shot 2022-02-08 at 1 10 57 PM](https://user-images.githubusercontent.com/25125247/153076258-610a7f64-2849-4f23-ab36-d05ed04309e7.png)
Core:
![Screen Shot 2022-02-08 at 1 11 24 PM](https://user-images.githubusercontent.com/25125247/153076322-3989129e-a79f-4803-b865-775809481b72.png)

![Screen Shot 2022-02-08 at 1 11 45 PM](https://user-images.githubusercontent.com/25125247/153076386-402aab65-b5b6-4707-b425-d9fe623ba5b5.png)
![Screen Shot 2022-02-08 at 1 12 10 PM](https://user-images.githubusercontent.com/25125247/153076439-d8ee84df-2a4e-4ad9-8838-55facef12e69.png)
